src/Provider/YamlConfigServiceProvider.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Islandora\Crayfish\Commons\Provider;
+
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Symfony\Component\Yaml\Yaml;
+use InvalidArgumentException;
+
+class YamlConfigServiceProvider implements ServiceProviderInterface
```

Comment:
Wondering if using the Symfony ConfigurationInterface might not be preferable? Allows us to easily set defaults and do configuration validation?

http://symfony.com/doc/current/components/config/definition.html

See also @acoburn's use in static-ldp.
https://github.com/trellis-ldp/static-ldp/blob/master/src/TrellisConfiguration.php